src/utils/viewer/viewer.cpp

```diff
@@ -359,7 +360,11 @@ void Viewer::drawEvent() {
 void Viewer::viewportEvent(ViewportEvent& event) {
   GL::defaultFramebuffer.setViewport({{}, framebufferSize()});
   renderCamera_->getMagnumCamera().setViewport(event.windowSize());
-}
+
+  int width = event.windowSize()[0];
+  int height = event.windowSize()[1];
+  renderCamera_->setProjectionMatrix(width, height, znear_, zfar_, hfov_);
+}  // namespace
```

Comment:
Why the `// namespace`?

Reply:
Oh, this is because the "vim" sometimes would automatically add the brackets for the user (totally unnecessarily), and if the user did not pay attention to it, and applied the clang-format, such "wrong" comments would be auto-added.